Documentation/cmdref/cilium-operator.md

```diff
@@ -67,6 +68,7 @@ cilium-operator [flags]
       --synchronize-k8s-services                Synchronize Kubernetes services to kvstore (default true)
       --unmanaged-pod-watcher-interval int      Interval to check for unmanaged kube-dns pods (0 to disable) (default 15)
       --update-ec2-apdater-limit-via-api        Use the EC2 API to update the instance type to adapter limits
+      --user-assigned-identity-name string      Name of the user assigned identity used to auth with the Azure API
```

Comment:
Maybe we should add a prefix `azure` to this flag since it currently does not have any reference to azure on its name.

Reply:
I'm down for that -- was actually thinking abt it.